terraform/shared/ecr.tf

```diff
@@ -6,10 +6,7 @@ resource "aws_ecr_repository" "items" {
   }
 }
 
-# This is in the identity account as it is deployed there
 resource "aws_ecr_repository" "requests" {
```

Comment:
I have manipulated the terraform state already to make this work. I've removed the identity ECR repo from TF state and will need to manually delete it later after we have demonstrated deployments work using the catalogue account ECR repo.